Dockerfile

```diff
@@ -1,3 +1,12 @@
+# syntax=docker/dockerfile:1
```

Comment:
seems like we do not use `start-cosmopark` from this repo, our cosmopark configuration moved to integration tests. May be we can get rid of the dead code?

Reply:
this actually makes sense

Reply:
we'll do that is a separate pr